faststream/nats/subscriber/asyncapi.py

```diff
@@ -194,3 +224,11 @@ class AsyncAPIDefaultSubscriber(AsyncAPISubscriber, DefaultHandler):
 
 class AsyncAPIBatchSubscriber(AsyncAPISubscriber, BatchHandler):
     """Batch-message consumer with AsyncAPI methods."""
+
+
+class AsyncAPIKvWatchSubscriber(AsyncAPISubscriber, KvWatchHandler):
```

Comment:
AsyncAPI schema for these classes should be empty. Please, override such methods for them.